Objects/longobject.c

```diff
@@ -37,6 +37,7 @@ medium_value(PyLongObject *x)
 #define IS_SMALL_UINT(ival) ((ival) < _PY_NSMALLPOSINTS)
 
 #define _MAX_STR_DIGITS_ERROR_FMT "Exceeds the limit (%d) for integer string conversion: value has %zd digits"
+#define _MAX_STR_DIGITS_ERROR_FMT2 "Exceeds the limit (%d) for integer string conversion"
```

Comment:
Here I've dropped the actual number of digits for the int->str conversion direction, since we can't compute that effectively up front. This macro could possibly use a better name.

Reply:
We could approximate a minimum and maximum, but agreed, it isn't worth doing.  I renamed the constants.